node/runtime/src/lib.rs

```diff
@@ -78,6 +79,14 @@ fn account_id_to_bytes(col: &[u8], account_key: &AccountId) -> Vec<u8> {
     key
 }
 
+// TODO(#362): Maybe need to replace Receipt nonce with the Receipt hash.
+// It's actually might be fine as well, since nonce is unique.
```

Comment:
I think it's fine. Should we remove the issue then?

Reply:
Depends on the future implementation of block explorer